Android/app/src/main/java/com/example/kahye/common/Adapter.java

```diff
@@ -11,21 +11,32 @@
 import android.widget.ImageButton;
 import android.widget.TextView;
 
-import java.text.SimpleDateFormat;
-import java.util.Date;
+import com.example.kahye.common.api_interface.ApiInterface;
```

Comment:
Why are these all in com.example.kahye? Can we move this to proper packages?

Reply:
We will find out how to change to packages.